float32 cpu_percent
float32 mem_percent
ProcessStatus[] nodes
